tests/Unit/CommentTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\Comment;
+use App\Models\JobPoster;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class CommentTest extends TestCase
+{
+    use RefreshDatabase, WithFaker;
+
+    /**
+     * Test one-to-many relationship between job poster and comments
+     *
+     * @return void
+     */
+    public function testCommentBelongsToAJobPoster()
+    {
+        $job_poster = factory(JobPoster::class)->create();
+        $comment = factory(Comment::class)->make(['job_poster_id' => $job_poster->id]);
+
+        $this->assertInstanceOf(JobPoster::class, $comment->job_poster);
+    }
+
+    /**
+     * Test one-to-many relationship between users and comments
+     *
+     * @return void
+     */
+    public function testCommentBelongsToAUser()
+    {
+        $user = factory(User::class)->create();
+        $comment = factory(Comment::class)->make(['job_poster_id' => $user->id]);
+
+        $this->assertInstanceOf(User::class, $comment->user);
+    }
```

Comment:
@yonikid15 I don't think we should bother writing tests like these. Eloquent model relationships are a key part of the Laravel framework, we kind of need to trust that they work. Writing and running these tests might be a misuse of our time.

Reply:
Okay will keep this in mind for next time. I was following this tutorial https://dev.to/tonyfrenzy/part-2-testing-model-relationships-in-laravel-basic-3a44 . I think the goal was to test that the relationships exists and set correctly, so if someone makes any changes it to the relationships it will break.

Reply:
That's fair. Still, I wouldn't bother next time.